### PR TITLE
Change loader for 5k tables test - newest loader doesn't work for this test

### PR DESCRIPTION
**Pull request by yarongilor**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
this is also a partial workaround for:
https://github.com/scylladb/scylla-cluster-tests/issues/3257

**Review by amoskong — APPROVED** [review]
LGTM

File: test-cases/longevity/longevity-5000-tables.yaml [hunk 1]
```diff
@@ -29,6 +29,8 @@ regions_data:
     ami_id_loader: 'ami-0b94f0e897d884b1b'
   eu-west-1:
     ami_id_loader: 'ami-0bf19545bc7bc9e1f'
+  eu-north-1:
+    ami_id_loader: 'ami-036150abbde18c61a'
```
Comment (roydahan) [inline test-cases/longevity/longevity-5000-tables.yaml:33]: AFAIR and now Amos approved it, this loader AMI is the older version of the loader that has Java 8 and proved to work for this longevity.

You need to find the same version of loader also for the other regions and update them as well.

also, next to them add a comment with the reason we use this loader.
e.g.:
```suggestion
    ami_id_loader: 'ami-036150abbde18c61a'    # Loader AMI v5 (I made this number up) - with java 8 - last loader works for this test.
```
```suggestion
    ami_id_loader: 'ami-036150abbde18c61a'
```

**Review by roydahan — CHANGES_REQUESTED** [review]

File: test-cases/longevity/longevity-5000-tables.yaml (hunk 1 above)
Comment (yarongilor) [reply test-cases/longevity/longevity-5000-tables.yaml:33]: OK, so the plan is covering all 6 supported regions for this loader AMI.
i think in this case we also have to add it similarly to the multi-keyspace yaml.

File: sdcm/logcollector.py [hunk 2]
```diff
@@ -782,6 +782,8 @@ class LoaderLogCollector(LogCollector):
                 search_locally=True),
         FileLog(name='scylla-bench-l*.log',
                 search_locally=True),
+        FileLog(name='kcl-l*.log',
+                search_locally=True),
```
Comment (roydahan) [inline sdcm/logcollector.py:786]:
```suggestion
```

**Review by roydahan — APPROVED** [review]

**Pull request merged**